calico/felix/felix.py

```diff
@@ -242,6 +243,13 @@ def dump_top_level_actors(log):
             _log.warning("Unable to install diag dump handler")
             pass
 
+        # Count cluster existence
+        if config.USAGE_COUNT:
+            calico_version = pkg_resources.require("calico")[0].version
```

Comment:
Why not move the version lookup inside the usage_count function, so that it can be common?